docs/reference/modules/terraform-aws-asg/asg-instance-refresh/asg-instance-refresh.md

```diff
@@ -73,6 +76,138 @@ The Terraform [instance_refresh](https://registry.terraform.io/providers/hashico
 *   Auto Scaling Groups support up to one active instance refresh at a time. When this resource is updated, any existing refresh is cancelled.
 *   Depending on health check settings and group size, an instance refresh may take a long time or fail. The Terraform `instance_refresh` resource does not wait for the instance refresh to complete.
 
+## Sample Usage
+
+<ModuleUsage>
+
+```hcl title="main.tf"
```

Comment:
NIT: I think you are aware of this, but rendering a tab here with "Terraform" and "Terragrunt" options would be great. I'm guessing you have a ticket to track that as follow-up work somewhere?

Reply:
Yes, in fact - that's exactly what's there now just the "Terragrunt" section only has a "Coming soon!" message in it. There's already a [JIRA](https://gruntwork.atlassian.net/browse/CORE-570) to track the work to add in a Terragrunt example.